lib/index.js

```diff
@@ -102,6 +105,15 @@ module.exports = function (gulp, options) {
     runWithDependencyProcess(runMocha, _.flattenDeep([options.unitTestFiles, options.integrationTestFiles]), cb);
   });
 
+  gulp.task('test-junit', 'Run all the tests with code coverage and output the results as JUNIT. (Specify ', ['lint-junit'], function (cb) {
```

Comment:
Should there also be a way to just run unit tests with the junit reporter? Or maybe just documenting `reporter` and `mocha.reporterOptions` should be sufficient instead of adding a new task to run with differing options.